render.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+documentation_folder="Documentation"
+index_file="$documentation_folder/Index.rst"
+readme_rst_file="README.rst"
+readme_md_file="README.md"
+output_folder="output"
```

Comment:
Currently, I don't like the folder name "output" at all. What output? Of a test? Of a custom script from an extension? OF the docs? Can we keep the well-know Documentation-GENERATED-temp?

Reply:
Yes, why not